install.sh

```diff
@@ -33,7 +33,12 @@ fi
 #     Version of the rke2 RPM release to install.
 #     Format would be like "1.el7" or "2.el8"
 #
+#   - INSTALL_RKE2_TAR_DIR
```

Comment:
I'd prefer this be called something like `INSTALL_RKE2_PREFIX` or `INSTALL_RKE2_ROOT`

Reply:
Or possibly `INSTALL_RKE2_TAR_PREFIX`

Reply:
I wanted to make it clear that it was `tar` specific since it does not relocate RPM installs.